utils/src/test/java/software/amazon/awssdk/utils/BinaryUtilsTest.java

```diff
@@ -159,4 +159,46 @@ public void testCopyBytesFrom_DirectByteBuffer_Idempotent() {
         assertTrue(partial1.length == 3);
         assertTrue(Arrays.equals(new byte[] {2, 3, 4}, partial1));
     }
+
+    @Test
+    public void testCopyRemainingBytesFrom_nullBuffer() {
+        assertNull(BinaryUtils.copyRemainingBytesFrom(null));
```

Comment:
Can we use assertJ for all our new tests? I get that you don't want to refactor all the existing tests, but at least for new tests.

Reply:
Yeah, I didn't want to start mixing them within the same test class, but I'm fine either way.